src/backend/plaf.rs

```diff
@@ -121,6 +122,39 @@ impl<F: PrimeField<Repr = [u8; 32]>, TraceArgs, StepArgs: Clone>
             plaf.lookups.push(plaf_lookup);
         }
 
+        if !self.circuit.exposed.is_empty() {
+            // Public column not pulled from Chiquito ir, because it's not stored anywhere.
+            // Therefore, we create a Plaf public column from scratch.
+            let plaf_public = ColumnPublic::new(String::from(
+                "exposed forward signal values in first step instance",
+            ));
+            plaf.columns.public.push(plaf_public);
+        }
+
+        for (index, (c_column, rotation)) in self.circuit.exposed.iter().enumerate() {
+            let public_column = pColumn {
```

Comment:
@qwang98 shouldn't this be the same as plaf_public?

Reply:
They are actually different structs.

plaf_public is of `ColumnPublic` type, which is stored in the `Plaf` data struct.
```
pub struct ColumnPublic {
    pub name: String,
    pub aliases: Vec<String>,
}
```

public_column is of `Column` type, which is used for expressing copy constraints in this context.
```
pub struct Column {
    pub kind: ColumnKind,
    pub index: usize,
}
```